src/areaSearch/components/AreaSearchApplicationListPage.js

```diff
@@ -365,6 +370,11 @@ class AreaSearchApplicationListPage extends PureComponent<Props, State> {
     }
 
     searchQuery.limit = LIST_TABLE_PAGE_SIZE;
+
+    if (searchQuery.service_unit === undefined && userActiveServiceUnit) {
```

Comment:
In Abstract there are 3 tabs (own work, own unit's work, and all area searches). Does this functionality allow user to see all area searches from all service units by default?